src/app/shared/services/openWeatherMaps.js

```diff
@@ -0,0 +1,42 @@
+export default class OpenWeatherMaps {
+    constructor($http, fiveDayForecast) {
+        this.$http = $http;
+        this.fiveDayForecast = fiveDayForecast;
+    }
+
+    getActualForecastCity(city) {
+        const URL = 'https://api.openweathermap.org/data/2.5/weather';
+
+        const request = {
+            method: 'GET',
+            url: URL,
+            params: {
+                q: city,
+                units: 'metric',
+                lang: 'en',
+                appid: process.env.OPENWEATHERMAP_API_KEY
+            }
+        };
+
+        return this.fiveDayForecast = this.$http(request);
```

Comment:
You can use the shortcut version:
https://code.angularjs.org/1.5.11/docs/api/ng/service/$http#get